components/post_view/channel_intro_message/index.ts

```diff
@@ -2,58 +2,28 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
-
 import {bindActionCreators, Dispatch} from 'redux';
 
-import {getConfig} from 'mattermost-redux/selectors/entities/general';
-import {getCurrentChannel, getDirectTeammate} from 'mattermost-redux/selectors/entities/channels';
-import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
-import {getProfilesInCurrentChannel, getCurrentUserId, getUser, getTotalUsersStats as getTotalUsersStatsSelector} from 'mattermost-redux/selectors/entities/users';
+import {getCurrentChannel} from 'mattermost-redux/selectors/entities/channels';
+import {getTotalUsersStats as getTotalUsersStatsSelector} from 'mattermost-redux/selectors/entities/users';
 import {get} from 'mattermost-redux/selectors/entities/preferences';
-
-import {getChannelIntroPluginComponents} from 'selectors/plugins';
-
 import {getTotalUsersStats} from 'mattermost-redux/actions/users';
-
 import {Preferences} from 'utils/constants';
-import {getDisplayNameByUser} from 'utils/utils';
-import {getCurrentLocale} from 'selectors/i18n';
-
 import {GlobalState} from 'types/store';
-
 import {GenericAction} from 'mattermost-redux/types/actions';
 
 import ChannelIntroMessage from './channel_intro_message';
 
 function mapStateToProps(state: GlobalState) {
-    const config = getConfig(state);
-    const enableUserCreation = config.EnableUserCreation === 'true';
-    const isReadOnly = false;
-    const team = getCurrentTeam(state);
     const channel = getCurrentChannel(state) || {};
-    const teammate = getDirectTeammate(state, channel.id);
-    const creator = getUser(state, channel.creator_id);
-    const boardComponent = getChannelIntroPluginComponents(state).find((c) => c.pluginId === 'focalboard');
-
-    const usersLimit = 10;
-
     const stats = getTotalUsersStatsSelector(state) || {total_users_count: 0};
+    const usersLimit = 10;
```

Comment:
is this intentionally hard-coded? Should we probably create a `Constant` or a setting for it?

Reply:
It is a magical number, but I'm not sure what to do with it. It determines if the number fo users on the team is less that `10`. The message will appear as below with an invite link to add members to the team.
<img width="602" alt="image" src="https://user-images.githubusercontent.com/7575921/190281899-131b7595-39a7-4fa5-a292-526ad12a5cf8.png">

Reply:
I'm 0/5 on if it should change, but would prefer we create a ticket if we want to investigate adding it as a setting.

Reply:
well ... if its only to determine if we should show the button to invite members to a team we can just make it a constant IMO